root/io/datamodelevolution/00/test2.cxx

```diff
@@ -39,17 +39,32 @@ void test2() {
    c->a.Print();
 
    TTree *tree; f->GetObject("tree",tree);
+   cout << "TTree::Scan object branch\n";
```

Comment:
Maybe you need a `flush` here to fix the sporadic test failure? Kinda strange...

Reply:
fixed.